api/app/routers/hfi_calc.py

```diff
@@ -42,7 +81,7 @@ async def get_daily_view(response: Response,
     """ Returns daily metrics for each station code. """
     try:
         logger.info('/hfi-calc/daily')
-        response.headers["Cache-Control"] = "max-age=0"  # don't let the browser cache this
+        response.headers["Cache-Control"] = no_cache
         valid_start_time, valid_end_time = validate_time_range(start_time_stamp, end_time_stamp)
```

Comment:
I know you haven't made any changes to this line of code, but I just noticed it, and huh??? How is this not infinitely recursive - the method is calling itself ??????

Reply:
Hmm I don't see where any method is calling itself here. `validate_time_range` is not recursive.